src/components/Settings.vue

```diff
@@ -320,6 +320,14 @@
                           persistent-hint
                         ></v-switch>
                       </v-flex>
+                      <v-flex xs12 v-if="gateway.hassDiscovery">
+                        <v-text-field
+                          v-model="gateway.entityTemplate"
+                          label="Entity name template"
+                          persistent-hint
+                          hint="Template which generates entity names<br>1"
+                        ></v-text-field>
```

Comment:
Add a legend here, you can use a `<p>` tag

Also this should be set to default or the default should be specified somewhere

Reply:
I have taken care the default value is set in function if this is undefined, i did not know how to set default here
still making function to cover both existense of valueId and when undefined. once I test all possible scenarios I will push my change and then will look into this!